README.md

```diff
@@ -1,5 +1,130 @@
+
+2. INSERT
+
+### Legal Framework with Related Statistical Analysis 
+
+The use of location data as solution to address the spread of COVID-19 directly impacts interrelated but distinctly different terms under privacy law: 
+
+* Pseudonymization;
+* De-identification;
+* Anonymization; and
+* Aggregation.
+
+These terms are similar and often used interchangeably. The terms generally address the core idea of removing identifiers from data. However, with regulations and standards, each termcan be defined and used distinctinly differentlly. 
+
+#### Pseudonymization
+
+The term pseudonymization is most relevant within the European Union under the General Data Protection Regulation ("GDPR"). [1] The GDPR defines pseudonymization as: 
+
+> the processing of personal data in such a manner that the personal data can no longer be attributed to a specific data subject without the use of additional information, provided that such additional information is kept separately and is subject to technical and organisational measures to ensure that the personal data are not attributed to an identified or identifiable natural person [Art. 4(5)].
+
+Two key factors are inherent in this defition: (1) attribution and (2) additional information. Based on this defintion, it appears that so long as the "additional information" that can be use to identify the information is removed and separated from the non-identifer information, the data converts from personal data to data that does not fall under the GDPR.
+
+Based on this, our project generated the following generalized defintion for pseudonymization:
+
+> the processing of personal information in a manner that renders the personal information no longer attributable to a specific consumer without the use of additional information.
+
+#### De-Identification
+
+The term de-identification is used within US-based cybersecurity and data privacy concepts. The National Institutes of Standards and Technology ("NIST") relies on this concept for removing identifiers from personal information, defining de-identification as:
+
+> general term for any process of removing the association between a set of identifying data and the data subject [2].
+
+By removing this association between the identifier and the information, information becomes de-identified, thereby reducing certian privacy concerns. Additional uses of the term "de-idenfitication" can be found in the Health Insurance Portability and Accountability Act (HIPAA) and the California Consumer Privacy Act of 2018. 
+
+Based on these sources, our project generated the following generalized definition for "de-identification":
+
+> Information that cannot reasonably identify, relate to, describe, be capable of being associated with, or be linked, directly or indirectly, to a particular individual.
+
```

Comment:
Strong.   But, what if such information relates to 2 individuals?  Is that still de-identified?  I'm focusing on the phrase "a particular individual"

Reply:
Addressed